status-im/status-react
# implemented ui for community-link card

fixes #14123 

### Summary

[comment]: # (Summarise the problem and how the pull)
created a reusable component for community-link card

#### Platforms


- Android
- iOS
- macOS
- Linux
- Windows

### Steps to test


- Open Status
- Open community
- Should see a community card for decentraland

status: ready
